main.go

```diff
@@ -43,33 +44,30 @@ func addMember(c *gin.Context) {
 }
 
 func main() {
-	args := parseArguments()
+	args := parseConfig()
 
 	router := gin.Default()
+	api := router.Group("/api")
 
-	members := router.Group("/members", security.AuthorizeJWTMiddleware())
+	corsConfig := cors.DefaultConfig()
+	corsConfig.AllowOrigins = []string{"http://localhost:5173"}
+	corsConfig.AllowCredentials = true
+	api.Use(cors.New(corsConfig))
+
+	members := api.Group("/members", security.AuthorizeJWTMiddleware())
 	members.GET("/", getMembers)
 	members.POST("/", addMember)
 
 	db := createDB(args.Debug)
 	userDB := security.NewUserDB(db)
 	webAuthNService := security.NewWebAuthNService(userDB, args.Origin, args.Domain)
 
-	router.GET("/register/:username", webAuthNService.StartRegister)
-	router.POST("/register/:username", webAuthNService.FinishRegistration)
-
-	router.GET("/login/:username", webAuthNService.StartLogin)
-	router.POST("/login/:username", webAuthNService.FinishLogin)
-	router.POST("/logout", webAuthNService.Logout)
-
-	router.LoadHTMLGlob("templates/*.html")
+	api.GET("/register/:username", webAuthNService.StartRegister)
```

Comment:
can this be moved to the service creation or somewhere else? Does it make sense to move it?

Reply:
Keep it here, clean up the main.go file in the ticket #4